### PR TITLE
feat: add downvoted property to gql post

### PR DESCRIPTION
**Pull request by capJavert**
Add `downvoted` property to gql post and boot to be used in companion.

### CODE REVIEW

File: src/routes/boot.ts [hunk 1]
```diff
@@ -394,6 +394,7 @@ const COMPANION_QUERY = parse(`query Post($url: String) {
           author {
             id
           }
+          downvoted
```
Comment (rebelchris) [inline src/routes/boot.ts:397]: Not sure, but I thought we also had a specific companion boot test that should be affected.

**Review by rebelchris — APPROVED** [review]
One small comment, but looks great, nice job on the test.

File: src/routes/boot.ts (hunk 1 above)
Comment (idoshamun) [reply src/routes/boot.ts:397]: Please validate it before merging :)

Comment (capJavert) [reply src/routes/boot.ts:397]: @rebelchris this one 92972601b44b0444789e6d45d00d305be11b1256?

Comment (rebelchris) [reply src/routes/boot.ts:397]: Ah yes, sorry thought that was the regular boot (sometimes hard to see in the PR screen 😅)

**Pull request merged**